src/Compilers/Core/Portable/PortableShim.cs

```diff
@@ -413,6 +413,11 @@ internal static class Encoding
                 .GetTypeInfo()
                 .GetDeclaredMethod(nameof(GetEncoding), paramTypes: new[] { typeof(int) })
                 .CreateDelegate<Func<int, System.Text.Encoding>>();
+
+            internal static readonly Func<System.Text.Encoding> Default = Type
```

Comment:
@AlekseyTs the best alternative I'm aware of is `CultureInfo.CurrentUICulture.TextInfo.ANSICodePage`